src/gui/mzroll/mainwindow.cpp

```diff
@@ -2215,7 +2225,7 @@ int MainWindow::loadMetaCsvFile(string filename){
     int lineCount=0;
     map<string, int>header;
     vector<string> headers;
-    static const string allHeadersarr[] = {"sample", "set", "injection order"};
+    static const string allHeadersarr[] = {"sample", "set", "cohort," "scaling", "injection order"};
```

Comment:
Typo. Comma is inside the quotes.

Reply:
Corrected (locally). Thanks for pointing out.